README.md

```diff
@@ -1,9 +1,7 @@
 .NET Compiler Platform ("Roslyn") Analyzers
```

Comment:
If we think there's moving forward value here, what I can do is merge your tool with ours and add the tools project to this repo. Afterwards, people could use the tool to extract docs, as per your scenario. Or, they could pass the tool a specially-formatted TSV file in order to produce starting project infrastructure for a new analyzer. Your functionality is likely to be more highly used moving forward (since we'll want to regen the docs anytime a rule is added).